README.md

```diff
@@ -84,25 +91,25 @@ The Car Racing environment features a physics model that affects the behaviour o
-* Monitor: one of Gym's provided wrappers. It provides an easy way to record the output of the environment to a video file.
-* GrayScaleObservation: another provided wrapper; it transforms RGB images to monochrome. Useful for reducing dimensionality in cases where the additional RGB info does not provide useful info compared to black and white images.
-* FrameStack: another provided wrapper; FrameStack allows us to "stack" frames (states) in order to create a "mini-batch" of sorts for more efficient training.
-* FrameSkipper: an original wrapper; FrameSkipper allows us to "skip" frames: instead of choosing an action for each frame, we use the same action for all of the skipped frames. This allows us to reduce the amount of actions we need to calculate.
-* EarlyStop: an original wrapper; when used, the environment will output `done = True` in additional circunstances besides the default ones, such as getting a negative average reward. This allows us to stop the execution early and train with more episodes.
+* **Monitor**: one of Gym's provided wrappers. It provides an easy way to record the output of the environment to a video file.
+* **GrayScaleObservation**: another provided wrapper; it transforms RGB images to monochrome. Useful for reducing dimensionality in cases where the additional RGB info does not provide useful info compared to black and white images. Therefore, it reduces the number of channels from 3 (RGB) to 1 (Grayscale).
+* **FrameStack**: another provided wrapper; FrameStack allows us to "stack" frames (states) in order to create a "mini-batch" of sorts for more efficient training. Also giving more context to the model (increase of the perceptive field).
+* **FrameSkipper**: an original wrapper; FrameSkipper allows us to "skip" frames: instead of choosing an action for each frame, we use the same action for all of the skipped frames. This allows us to reduce the amount of actions we need to calculate.
+* **EarlyStop**: an original wrapper; when used, the environment will output `done = True` in additional circumstances besides the default ones, such as getting a negative average reward. This allows us to stop the execution early and train with more episodes.
 
 There is also the issue of defining what "consistently" means when trying to get a reward of "consistently more than 900 points". We settled on calculating a *running reward* that accumulates the previously obtained rewards and calculates an average of sorts that represents the reward you can expect from the model at a specific stage of training, which is calculated as `running_reward = 0.05 * episode_reward + (1 - 0.05) * running_reward`, where `episode_reward` is the reward we obtain for each lap attempt.
 
 For all of our experiments, the chosen optimizer was Adam, since it seems to be the default optimizer for pretty much any task. We did not experiment with additional optimizers.
```

Comment:
Shouldn't we add a more scientific sentence for the Adam? hahahahaha
Like adding a reference to theory slides where it compares both of them. Or something like that?

Reply:
I honestly can't think up of anything. Optimizers aren't the focus of this project, so I would just let it be.